app/build.gradle

```diff
@@ -33,56 +33,62 @@ android {
+    compile "com.scottyab:rootbeer-lib:0.0.6"
+    compile group: "com.google.code.gson", name: "gson", version: "2.7"
 
-    compile 'com.jakewharton:butterknife:8.8.1'
-    annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
+    compile "com.jakewharton:butterknife:8.8.1"
+    annotationProcessor "com.jakewharton:butterknife-compiler:8.8.1"
 
-    compile 'com.google.dagger:dagger:2.11'
-    compile 'com.google.dagger:dagger-android:2.11'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.11'
-    annotationProcessor 'com.google.dagger:dagger-android-processor:2.11'
+    compile "com.google.dagger:dagger:$dagger_version"
+    compile "com.google.dagger:dagger-android:$dagger_version"
+    compile "com.google.dagger:dagger-android-support:$dagger_version"
+    annotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
+    annotationProcessor "com.google.dagger:dagger-android-processor:$dagger_version"
 
-    compile 'net.zetetic:android-database-sqlcipher:3.5.7@aar'
+    compile "net.zetetic:android-database-sqlcipher:3.5.7@aar"
 
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:2.10.0'
-    testCompile 'org.json:json:20160810'
+    testCompile "junit:junit:4.12"
+    testCompile "org.mockito:mockito-core:2.10.0"
+    testCompile "org.json:json:20160810"
     testCompile "org.robolectric:robolectric:3.4.2"
 
-    androidTestCompile 'org.mockito:mockito-android:2.10.0'
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
-        exclude group: 'com.android.support', module: 'support-annotations'
+    androidTestCompile "org.mockito:mockito-android:2.10.0"
+    androidTestCompile("com.android.support.test.espresso:espresso-core:2.2.2", {
+        exclude group: "com.android.support", module: "support-annotations"
     })
-    androidTestCompile('com.android.support.test.espresso:espresso-contrib:2.2.2') {
-        exclude group: 'com.android.support', module: 'support-annotations'
-        exclude group: 'com.android.support', module: 'appcompat-v7'
-        exclude group: 'com.android.support', module: 'support-v4'
-        exclude group: 'com.android.support', module: 'design'
-        exclude module: 'recyclerview-v7'
+    androidTestCompile("com.android.support.test.espresso:espresso-contrib:2.2.2") {
+        exclude group: "com.android.support", module: "support-annotations"
+        exclude group: "com.android.support", module: "appcompat-v7"
+        exclude group: "com.android.support", module: "support-v4"
+        exclude group: "com.android.support", module: "design"
+        exclude module: "recyclerview-v7"
     }
-    androidTestAnnotationProcessor 'com.google.dagger:dagger-compiler:2.11'
-    androidTestAnnotationProcessor 'com.google.dagger:dagger-android-processor:2.11'
+    androidTestAnnotationProcessor "com.google.dagger:dagger-compiler:2.11"
+    androidTestAnnotationProcessor "com.google.dagger:dagger-android-processor:2.11"
```

Comment:
is there a big difference between single quote and double quote in gradle?

Reply:
Single quote can allow variable `androidTestAnnotationProcessor "com.google.dagger:dagger-compiler:$some_variable"`